### PR TITLE
Css a scss

### PR DESCRIPTION
**Pull request by JaqKent**

### CODE REVIEW

File: css/_add-item.scss [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+      color: $blue;
+      font-family: "Titillium Web", sans-serif;
+      font-size: 20px;
+    }
+    &:-moz-placeholder {
+      /* Firefox 18- */
+      color: $blue;
+      font-family: "Titillium Web", sans-serif;
+      font-size: 20px;
+    }
```
Comment (nathan-hyan) [inline css/_add-item.scss]: This can be deleted since SCSS creates this automatically for you

File: css/_content.scss [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+  i.de {
+    position: absolute;
+    font-size: 25px;
+    right: 15px;
+    top: 10px;
+  }
+  i.de:hover {
+    color: $red;
+    cursor: pointer;
+  }
```
Comment (nathan-hyan) [inline css/_content.scss]: Not sassy enough

File: css/_content.scss [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+  i.co {
+    position: absolute;
+    font-size: 25px;
+    padding-left: 5px;
+    left: 15px;
+    top: 10px;
+  }
+  i.co:hover {
+    cursor: pointer;
+  }
```
Comment (nathan-hyan) [inline css/_content.scss]: Not sassy enough

File: css/_content.scss [hunk 4]
```diff
@@ -0,0 +1,62 @@
+.content {
+  width: $box-width;
+  height: 350px;
+  max-height: 350px;
+  background-color: $white;
+  overflow: auto;
+
+  .content::-webkit-scrollbar {
+    display: none;
+  }
```
Comment (nathan-hyan) [inline css/_content.scss]: Not sassy enough, remember the use of `&:`

**Review by nathan-hyan — CHANGES_REQUESTED** [review]
Good job! some comments

File: css/_add-item.scss (hunk 1 above)
Comment (JaqKent) [reply css/_add-item.scss]: done

File: css/_content.scss (hunk 3 above)
Comment (JaqKent) [reply css/_content.scss]: done

File: css/_content.scss (hunk 4 above)
Comment (JaqKent) [reply css/_content.scss]: done

File: css/_content.scss (hunk 2 above)
Comment (JaqKent) [reply css/_content.scss]: done

**Review by nathan-hyan — APPROVED** [review]
🥇

**Pull request merged**